components/public-api-server/pkg/server/server.go

```diff
@@ -45,10 +45,11 @@ func Start(logger *logrus.Entry, version string, cfg *config.Configuration) erro
 	}
 
 	dbConn, err := db.Connect(db.ConnectionParams{
-		User:     os.Getenv("DB_USERNAME"),
-		Password: os.Getenv("DB_PASSWORD"),
-		Host:     net.JoinHostPort(os.Getenv("DB_HOST"), os.Getenv("DB_PORT")),
-		Database: "gitpod",
+		User:         os.Getenv("DB_USERNAME"),
+		Password:     os.Getenv("DB_PASSWORD"),
+		Host:         net.JoinHostPort(os.Getenv("DB_HOST"), os.Getenv("DB_PORT")),
+		Database:     "gitpod",
+		CustomCACert: os.Getenv("DB_CUSTOM_CA_CERT"),
```

Comment:
Likewise here, I'd drop the custom.

Reply:
Yeah, good point. So far I have been torn we because I'd have to partially retest. But I think you're right. :100:

Reply:
Ok. Done, and re-tested: works.